server/index.js

```diff
@@ -50,7 +50,7 @@ app.use(compress())
 
 app.use(function (req, res, next) {
   // Force SSL
-  if (config.isProd && req.protocol !== 'https') {
+  if (config.isProd && req.headers['x-forwarded-proto'] !== 'https') {
```

Comment:
This will break when instant.io is not behind an http proxy. Can you change this to a check for either?